servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/ConnectionObserver.java

```diff
@@ -217,15 +230,15 @@ default void proxyConnectComplete(Object responseMsg) {
     interface SecurityHandshakeObserver {
 
         /**
-         * Callback when the handshake is failed.
+         * Callback when the handshake fails.
```

Comment:
While we are in here, I think there is an opportunity to clarify on the exact timing of the callbacks. Maybe it's because I am not a native speaker, but for me the "active" tense makes it a little unclear when the callback is invoked exactly. I am posting it here, but I think other callbacks have the same semantics (for example see the next two).

I would propose instead of `Callback when the handshake fails`: `Callback is executed after the handshake has failed`